c++/src/Compression.cc

```diff
@@ -199,13 +199,16 @@ namespace orc {
                           uint64_t blockSize,
                           MemoryPool& pool);
 
+    ~ZlibCompressionStream() { end(); }
```

Comment:
You need an override on this or the mac complains.

Reply:
Thanks!